### ISSUE TITLE
researcher-feedback: Lesson 5 teaches decontamination with a planted identical copy, the one case that cannot fail

### ISSUE THREAD

**Issue by jacobweiss2305** [whilehq/whileai-sdk#480]
## What I was trying to do

Decide whether I can trust a held-out set the SDK has decontaminated, before I quote a before/after number to anyone. I work on evals; the held-out set is the whole product.

## What I read

[Lesson 5, *The test the model never sees is the only score that counts*](https://docs.withwhile.com/learn/the-held-out-set.md). It is the page the docs point you to for contamination, and it teaches the check like this:

```python
# Plant one test row in the training set and see it caught.
kept, report = wai.decontaminate(train + holdout[:1], against=holdout)
```

```text
41 -> 40 training rows
contaminated: 1
```

and concludes:

> One planted copy, one row dropped. On a real set the count is rarely zero, and **every row it drops is a row that would have made the after score lie.**

## What I expected

That the demo would be representative of what the rule does on real data.

## What happened

The planted row is a **byte-identical copy**, which is the one input the rule catches 100% of the time. I measured it as a control: identical copy → caught 1.000, case-and-whitespace-only change → caught 1.000. So the lesson demonstrates the easy case and stops.

The sentence that follows is true but one-sided. "Every row it drops would have made the after score lie" says nothing about the rows it *doesn't* drop, and that is the number a reader needs.

I measured the rows it doesn't drop, using human-labelled paraphrase pairs as ground truth (QQP duplicates and PAWS, so the labels are not mine): a training row that asks the holdout's question **in different words**, carrying a task id from a different namespace — the ordinary case when rows come from another team, a vendor, or the Hub, where the `same_task` rule has nothing to match on.

`whileai==0.86`, 2,175 rows per seed, 1,200 real leaks, 975 genuinely-clean controls, intervals are `wai.pass_at(...).ci95`:

| rule | catches a real paraphrase leak | drops a clean row |
|---|---|---|
| **default** (`n=8, overlap=0.8`) | **0.094** [0.077, 0.113] | 0.044 [0.031, 0.058] |
| `overlap=0` (any shared 8-gram) | 0.260 [0.231, 0.288] | 0.182 [0.157, 0.207] |
| `n=5, overlap=0.8` | 0.168 [0.144, 0.192] | 0.083 [0.066, 0.101] |
| `n=5, overlap=0` | 0.558 [0.527, 0.592] | 0.422 [0.390, 0.453] |
| `n=3, overlap=0` | 0.928 [0.911, 0.944] | 0.869 [0.848, 0.890] |

Controls on every row of that table: identical copy 1.000, case-only 1.000, unrelated pair 0.03 — so the harness is sound and the 0.094 is the rule, not my setup.

**The default leaves ~91% of paraphrase contamination in the training set.** Loosening the rule buys recall almost exactly 1:1 with false positives, so there is no lexical setting that fixes it.

## What I wanted instead

Three things on that page, none of which need the numbers above to be exactly right:

1. **Say what the rule cannot see.** One sentence: word overlap does not see a paraphrase, so a rewritten holdout task survives decontamination. The `decontaminate()` docstring already says this honestly ("Word overlap does not see a paraphrase") and reports a real measurement. The lesson — the page a newcomer actually reads — says none of it.
2. **Mention `embedder=` exists.** The page describes only the 8-gram rule ("The library counts shared runs of eight words…"). `decontaminate()` has taken a semantic pass since #286 / #325, and a reader of Lesson 5 will never learn that.
3. **Plant a paraphrase, not a copy.** If the demo planted a reworded holdout row next to the identical one and showed one caught and one missed, the lesson would teach the actual state of the art instead of a reassurance.

As written, the page's demo is engineered to succeed, and a reader finishes it believing their held-out set is protected. Mine wasn't.

## Related

- #286 (closed) measured the same blind spot from the other direction and said the default should stay lexical "until someone does the threshold study". I ran that study; numbers to follow in a separate issue so this one stays about the page.
